test/tools/spec-runner/index.js

```diff
@@ -202,29 +210,31 @@ function generateTopologyTests(testSuites, testContext, filter) {
       if (!shouldRun) this.skip();
     };
 
-    describe(testSuite.name, function () {
-      beforeEach(beforeEachFilter);
-      beforeEach(() => prepareDatabaseForSuite(testSuite, testContext));
-      afterEach(() => testContext.cleanupAfterSuite());
-      for (const spec of testSuite.tests) {
-        const mochaTest = it(spec.description, async function () {
-          if (spec.failPoint) {
-            await testContext.enableFailPoint(spec.failPoint);
-          }
+    if (testSuite.name === 'fle2-CreateCollection') {
```

Comment:
this change is making it so only fle2-CreateCollection tests run from the fle test spec directory, is that intentional?